src/test/java/io/fabric8/maven/docker/access/ContainerCreateConfigTest.java

```diff
@@ -122,6 +123,7 @@ private String copyPropsToFile() throws IOException {
         Map<String,String> envMap = new HashMap<>();
         envMap.put("JAVA_OPTS", "-Xmx512m");
         envMap.put("TEST_SERVICE", "LOGGING");
+        envMap.put("TEST_HTTP_ADDR", "+${docker.container.consul.ip}");
```

Comment:
Please add also tests where the + sign is part of a value or when there is something in addition like in `+${docker.container.consul.ip}additional_data` (here, there should be no removal of the +)